CHANGELOG.md

```diff
@@ -1,3 +1,87 @@
+# 1.2.0 (2020-04-9)
+## AWARENESS
+* Enable nested edge by default
+* Move `NestedEdgeEnabled` settings out of `experimentalFeatures` section
+* MQTT Broker and MQTT Bridge settings are merged
```

Comment:
What is this referring to?

Reply:
>* MQTT Broker and MQTT Bridge settings are merged

I think this is not needed to be mentioned. From cx pov it was internal refactoring.